### DISCUSSION TITLE
[Q&A] 通过Vercel部署的，应该怎么设置环境变量中的access code，让这个密码生效？

### DISCUSSION THREAD

**Discussion by jackcheng321321** [lobehub/lobehub#813] [General Question] [answered]
通过Vercel部署的，应该怎么设置环境变量中的access code，让这个密码生效？

**Comment by canisminor1990** [comment, marked as answer]
`settings` - `environment-variables` - `Create new` - `Add another`

![image](https://github.com/lobehub/lobe-chat/assets/17870709/bd7a3ceb-7807-44f6-bf84-630e7dcf515b)
